mosquito/mapi.py

```diff
@@ -232,6 +247,30 @@ def set_motors(self, values):
 		"""
 		self.__motor_values = values
 		self._send_data(msppg.serialize_SET_MOTOR_NORMAL(*values))
+		
+	def set_voltage(self, voltage):
+		"""
+		Set the values of all motors in the specified order
+
+		:voltage: 4 value list with desired motor values in 
+		the range 0-1 being 1 maximum speed and 0 motor stopped.
+		:type values: float
+		:return: None
+		:trype: None
+		"""
+		self.__voltage = voltage
+		self._send_data(msppg.serialize_SET_BATTERY_VOLTAGE(voltage))
+	
+	def get_voltage(self):
+		"""
+		Get the orientation of the Mosquito
+
+		:return: Orientation of the Mosquito in radians
+		:rtype: tuple
+		"""
+		self._parser.set_GET_BATTERY_VOLTAGE_Handler(self.__handle_get_voltage)
```

Comment:
(I thought about this yesterday) I think it is better if we move the handler setters to the `__init__` method of the class. This way they are only set once and not every time the API method is called. I moved the rest of them yesterday. 

See:

https://github.com/BonaDrone/mosquito-API/blob/b3ad7bfbb0297d084f19f223ca000b45c53c8573/mosquito/mapi.py#L34-L37